fe/fe-core/src/main/java/com/starrocks/monitor/jvm/JvmStats.java

```diff
@@ -178,56 +187,6 @@ public Classes getClasses() {
         return classes;
     }
 
-    static final class Fields {
```

Comment:
looks like a non-related code clean-up, can submit a separate PR to remove these lines through a fast-track code review.

Reply:
em..., good advise, but I think no one would want to submit such a PR alone. After all, submitting a PR requires a lot of extra energy to maintain, such as CI, handling branch conflicts, etc. 
Some minor refactoring is acceptable in feature PRs, as no one will notice that the code is not very clean unless you are developing the feature.

Reply:
lol. That's quite messy and hated when sometimes later the fix is willing to be backport to historic branches, and if it is a different person to do the backport, it will be so hard to resolve the conflict what is the key change what is not.